packages/media-core/src/media_core/transcribe/backends/whisper_cpp.py

```
@@ -34,6 +34,48 @@ def _ensure_whispercpp():
 
 
+def _get_tokens(seg: Any) -> Any:
+    return getattr(seg, "tokens", None) or getattr(seg, "get", lambda k, d=None: d)("tokens", None)
+
+
+def _parse_token_word(tok: Any) -> Word | None:
+    try:
+        start = float(getattr(tok, "t_start", tok.get("t_start")))
+        end = float(getattr(tok, "t_end", tok.get("t_end")))
+        text = str(getattr(tok, "text", tok.get("text"))).strip()
+    except Exception as exc:  # pragma: no cover - defensive
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)